### PR TITLE
PRIME-2389 Obo Job Site changes

### PR DESCRIPTION
**Pull request by bergomi02**
Update validation rules

### CODE REVIEW

**Comment by neophyte57** [comment]
Getting this on happy path
1. Select HA Care Setting and Health Authority
2. No Licence
3. Enter Job Site information (Health Authority pre-set)
4. Save and Continuing to Self-Declaration

image

File: prime-angular-frontend/src/app/modules/enrolment/pages/obo-sites-page/obo-sites-page.component.ts [hunk 1]
```diff
@@ -107,6 +107,16 @@ export class OboSitesPageComponent extends BaseEnrolmentProfilePage implements O
     );
   }
 
+  public get showHASelectError(): boolean {
+    let result = false;
+    Object.values(this.healthAuthoritySites.value).forEach((s) => {
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/modules/enrolment/pages/obo-sites-page/obo-sites-page.component.ts:112]: Can you add a type to `s` variable?

Comment (bergomi02) [reply prime-angular-frontend/src/app/modules/enrolment/pages/obo-sites-page/obo-sites-page.component.ts]: A type is added.

**Review by neophyte57 — APPROVED** [review]
Works!

**Pull request merged**